Source/Schema.NET.Pending/Schema.NET.Pending.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Label="Build">
+    <TargetFrameworks>net5.0;netcoreapp3.1;netstandard2.0;net472;net461</TargetFrameworks>
+    <EmitCompilerGeneratedFiles>True</EmitCompilerGeneratedFiles>
+    <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
+    <IncludePendingSchemaObjects>True</IncludePendingSchemaObjects>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package">
+    <Product>Schema.NET.Pending</Product>
+    <Description>Schema.org objects (including pending types) turned into strongly typed C# POCO classes for use in .NET. All classes can be serialized into JSON/JSON-LD and XML, typically used to represent structured data in the head section of html page.</Description>
+    <PackageTags>Schema;.NET;Schema.org;Schema.NET;Structured Data;Google Structured Data</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup Label="Package References">
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="System.Memory" Version="4.5.4" />
+    <ProjectReference Include="../../Tools/Schema.NET.Tool/Schema.NET.Tool.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
+  </ItemGroup>
+
+  <ItemGroup Label="Package References (.NET Framework)" Condition="'$(TargetFramework)' == 'net461'">
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+  </ItemGroup>
+
+  <ItemGroup Label="Common Code">
+    <Compile Include="../Common/*.*" />
```

Comment:
Took me a second to get my head around this, so there are common files added as solution level items and we're using compiler magic to include them. That seems fine, although I do wonder if it would be nicer to leave the files in Schema.NET and link to them from Schema.NET.Pending or even introduce a common assembly Schema.NET.Common.

Reply:
I thought linking them from "Schema.NET" to "Schema.NET.Pending" seemed to imply an inheritance/importance between them where they are actually equals. (I know that might sound weird but it made sense in my head) 

I thought about using a common project too. It would mean less weirdness adding new common files (and how that interacts with the `.sln` format) but we would run into problems if we actually want that common project to compile. The file `PropertyValueSpecification.Partial.cs` relies on the generated source. We could either duplicate this file manually or only reference it in our individual projects but I think we would be exchanging one weirdness for another.

Reply:
Makes sense. I haven't had a chance to play with this PR in VS, is there any weirdness with compiler errors/warnings this way? I assume not.

Reply:
No errors or compiler warnings that I've seen. The only issue I've encountered is that solution folders don't act the same as folders in projects. If you right click -> add file to a solution folder, the file ends up in the root directory. If you right click -> add file to `Schema.NET` or `Schema.NET.Pending`, it would only add that file to that project.

Reply:
Yeah, hate solution folders for that reason and the fact that the sln file is still indecipherable.